src/plugins/RecommendNextComponents/RecommendNextComponents/models/graphconv/config.yaml

```diff
@@ -0,0 +1,106 @@
+bn:
+  eps: 1.0e-05
+  mom: 0.1
+cfg_dest: config.yaml
+dataset:
+  augment_feature: []
+  augment_feature_dims: []
+  augment_feature_repr: original
+  augment_label: ''
+  augment_label_dims: 0
+  cache_load: false
+  cache_save: false
+  dir: /mnt/design.r/spice/datasets/
```

Comment:
What does this signify? In  a `dockerized` setting, we have to mount this directory for this model to work?

Reply:
That is the data directory used when training the model. In the inference script, it should be ignored. I left it here because I wanted to make this model the same way I recommend others to make these models (and didn't want to instruct people to manually edit this large file generated from GraphGym).

Reply:
This is also why I edit the config in the inference file ([here](https://github.com/symbench/electric-circuits/pull/80/files#diff-26b20ab2b17357fba36e79d0c70305cecd8f6276a33942972488c6fa511b0752R30)) - I didn't want to add another step where users have to sift through the config when adding new models from GraphGym.